zgrab_schema.py

```diff
@@ -773,7 +773,7 @@
                 "client_to_server_languages":ListOf(String()),
                 "server_to_client_languages":ListOf(String()),
                 "first_kex_follows":Boolean(),
-                "reserved":Short(),
+                "reserved":Unsigned32BitInteger(),
```

Comment:
Does this _actually_ need 32 bits? This results logic results in a 64-bit integer.